jolt-core/src/main/java/com/bazaarvoice/jolt/templatr/OpMode.java

```diff
@@ -45,7 +49,7 @@ public boolean isApplicable( final List source, final int reqIndex ,  int origSi
             return super.isApplicable(source, reqIndex , origSize);
         }
     },
-    DEFAULTR {
+    DEFAULTR("_") {
```

Comment:
Swap the chars for Define and Default pls.

"_" underline rhymes with Define
"~" tends to mean an "approximation".  Overwrite and Define have rigid / clear semantis, Default is the most "ishy" of the ops, hence it seem like it is closer to an "approximation"